task/src/main/scala/aerospiker/task/Operation.scala

```diff
@@ -71,11 +69,11 @@ abstract class Aerospike(command: AsyncCommandExecutor) extends Operation with L
   }
 
   def puts[A](kvs: Map[String, A])(implicit encoder: Encoder[A]): Task[Seq[Throwable Xor String]] = {
-    Task.gatherUnordered {
+    Task.gatherUnordered({
       kvs map {
         case (k, v) => put(k, v)
       } toSeq
-    }
+    }, true)
```

Comment:
If `exceptionCancels` is set `false`, an exception tries to stop the other jobs immediately. Thus the parameter should be `false` as we want to try running all the jobs. And to explicitly call `Task.gatherUnordered(tasks, exceptionCancels)` instead of `Task.gatherUnordered(tasks)`, we have to pass the boolean value even though it is same as the default value.